src/sql/workbench/contrib/notebook/browser/cellViews/outputArea.css

```diff
@@ -9,7 +9,6 @@ output-area-component {
 output-area-component .notebook-output {
 	border-top-width: 0px;
 	user-select: text;
-	padding: 5px 20px 0px;
```

Comment:
The problem with removing this is that it becomes more difficult visually to distinguish between cells

Reply:
Agreed. @radentonev  I think 20px of padding-top to provide some space between the code and the output would do nicely.

Reply:
I'll restore the padding top. Previous value was padding-top is 5px. padding-left is 20px.